client/trino-cli/src/main/java/io/trino/cli/Console.java

```diff
@@ -246,9 +246,9 @@ private static void runConsole(
             while (!exiting.get()) {
                 // setup prompt
                 String prompt = PROMPT_NAME;
-                String schema = queryRunner.getSession().getSchema();
-                if (schema != null) {
-                    prompt += ":" + schema.replace("%", "%%");
+                Optional<String> schema = queryRunner.getSession().getSchema();
+                if (schema.isPresent()) {
+                    prompt += ":" + schema.get().replace("%", "%%");
```

Comment:
Nit: this can be simplified as (no change requested):
```suggestion
                prompt += schema.map(value -> ":" + value.replace("%", "%%"))
                		.orElse("");
```

Reply:
Done